### DISCUSSION TITLE
Assert if i received my Email Correctly in Gmail

### DISCUSSION THREAD

**Discussion by abdurraheemj** [seleniumbase/SeleniumBase#1244] [Q&A] [answered]
Hi Michael,
I am sending an Email from an application, I want to assert if my text is present in the email. Having trouble getting webElement. Any idea to achieve this? 
![Screen Shot 2022-03-08 at 12 03 36 PM](https://user-images.githubusercontent.com/69648204/157298525-a5a6dcdc-b449-4ad1-a191-e7c42332404b.png)

**Comment by mdmintz** [comment, marked as answer]
Open Dev Tools above that (Right-click + Inspect) and then find the element from there. Or try generating the script via the Recorder https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/recorder_mode.md
